sql-gremlin/src/main/resources/error-messages.properties

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright <2021> Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Should this have a -en suffix? Have you tested this with multiple languages, as usually one of the other reasons for using property bundles is allowing for easy localization.

Reply:
I think kyle is right, we missed that in the JDBC driver but I see it in our Bolt connector. Take a look here https://github.com/Bit-Quill/neptunebolt/tree/develop/src/main/resources @simonz-bq you will see the properties files have _en at the end.